contracts/governance/compatibility/GovernorCompatibilityBravo.sol

```diff
@@ -154,6 +154,9 @@ abstract contract GovernorCompatibilityBravo is IGovernorTimelock, IGovernorComp
             fullcalldatas[i] = bytes(signatures[i]).length == 0
                 ? calldatas[i]
                 : abi.encodePacked(bytes4(keccak256(bytes(signatures[i]))), calldatas[i]);
+            unchecked{
+                ++i;
+            }
```

Comment:
here you added `unchecked { ++i; }` but forgot to remove the `++i` from the for loop. This is really characteristic of the little mistake that you tend to do when trying to (over) optimize ...